examples/remote_procedure/toxicity_detection/webapp/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "bootstrap": "^4.6.0",
     "mephisto-task": "^2.0.1",
+    "mephisto-worker-experience": "file:../../../../packages/mephisto-worker-experience",
```

Comment:
Since we don't have `mephisto-worker-experience` published, this is the only way at the moment to link to the package. However, we probably want to change this to a version number (similar to how mephisto-task is in this folder). If we ship this as is, I think we should file a Github Issue mentioning to change this after the pkg is published so we don't lose track.

Another solution is to include these projects in the yarn workspace where yarn should link them up to the packages in the root project automatically.

Reply:
Makes sense 👍